### PR TITLE
Fix #19: Manager Drivers of Sample.UITest via Nuget

### LINKED ISSUE #19: Manager Drivers of Sample.UITest via Nuget
Now Drivers are *.exe files and when I ran Sample for first time, test failed cause driver was out of date.
With having them in Nuget we can easily update them by updating nuget package version to newest.

### PR DESCRIPTION
**Pull request by kmorcinek**

### CODE REVIEW

**Comment by kmorcinek** [comment]
I was too fast and FF was not tested properly, so please do not take any action.

**Comment by kmorcinek** [comment]
I have tested also on FF and created a bug #21 stating that the is already problem with FF and it does not affect this PR.

I am not able to run IE test on my current machine, so maybe somebody else can do it.

File: Src/MvcPages/Configuration/TelluriumConfigurationSection.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ public BrowserType Browser
             set { this[BrowserKey] = value;  }
         }
 
-        [ConfigurationProperty(DriversKey, DefaultValue = "Drivers", IsRequired = false)]
+        [ConfigurationProperty(DriversKey, DefaultValue = "", IsRequired = false)]
```
Comment (cezarypiatek) [inline Src/MvcPages/Configuration/TelluriumConfigurationSection.cs:24]: Please restore default value as it was before.  The reason is simple: when you add drivers directly to your project and use "copy to output" option VS keep the path in output directory (subfolder is copied as well). Keeping drivers in the main project's directory can cause a mess and confusion. When default value is ""Drivers" a options in config file driversPath="" should override default value correctly.

**Review by cezarypiatek — COMMENTED** [review]
I run sample tests locally and everything seems to work correctly.

**Comment by cezarypiatek** [comment]
This commit was so inspiring so I've created another issue #22. @kmorcinek what do you think about this idea?

**Comment by cezarypiatek** [comment]
This issue was solved with Tellurium.SeleniumDriverInstaller

**Pull request closed without merging**